http-gateway/web/src/containers/devices/_command-timeout-control.js

```diff
@@ -47,10 +47,10 @@ export const CommanTimeoutControl = ({
   )
 
   const units = Object.values(commandTimeoutUnits)
-    .filter(unit => unit !== NS)
-    .map(unit => ({
-      value: unit,
-      label: unit === INFINITE ? _(t.default) : unit,
+    .filter(_unit => _unit !== NS)
```

Comment:
Can you explain this change? I really don't see any reason why to change this for underscores :)

Reply:
Sonar ->” variable is already declared in the upper scope “

Reply:
Ah, I did not see it, sorry :)